tests/http/test_proxyagent.py

```diff
@@ -218,7 +218,7 @@ def test_https_request_via_proxy(self):
         agent = ProxyAgent(
             self.reactor,
             contextFactory=get_test_https_policy(),
-            https_proxy=b"proxy.com",
+            https_proxy=b"http://proxy.com",
```

Comment:
We'll want to ensure that the old way still works, so we should probably add an additional test.

Reply:
Agreed, I only changed it in one of two instances. (But didn't explicitly mention that anywhere... I'll add a comment into the tests!)